lib/twilio.rb

```diff
@@ -0,0 +1,28 @@
+require 'twilio-ruby'
+
+class Sender
+  attr_reader :client
+  def initialize
```

Comment:
Could use dependency injection on the twilio client here, and write a test to check that the create message is called